### PR TITLE
Sentry errors

### PR DESCRIPTION
**Pull request by AshakaE**
# Sentry review

_Returns null for entity not found, try block to keep indexer from throwing exceptions_

## How should this be tested?

1. getProfile
image
2. userById
image
3. tagById
image

## Linked issues

closes https://github.com/EveripediaNetwork/issues/issues/790

### CODE REVIEW

File: src/App/pinJSONAndImage/pin.resolver.ts [hunk 1]
```diff
@@ -24,11 +25,10 @@ class PinResolver {
 
   private errorHandler(val: any) {
     if (val.message) {
-      Logger.error(val.message)
-      throw new HttpException(val.message, HttpStatus.BAD_REQUEST)
-    } else {
-      return val
+      console.error('PIN JSON/IMAGE ERROR', val.message)
+      return true
```
Comment (kesar) [inline src/App/pinJSONAndImage/pin.resolver.ts:29]: shouldnt return false or null ?

File: src/App/tag.resolver.ts [hunk 2]
```diff
@@ -36,19 +35,11 @@ class TagResolver {
...
-
-    if (!tagId) {
-      throw new GqlError(
-        HttpStatus.NOT_FOUND,
-        `Tag with id '${id}' not found`,
-        ErrorTypes.NOT_FOUND,
-      )
-    }
-    return tagId
+    return tagId || null
```
Comment (kesar) [inline src/App/tag.resolver.ts:42]: is the || null needed? whats the return of typeORM if it doesnt exist tag?

File: src/App/userProfile.resolver.ts [hunk 3]
```diff
@@ -113,6 +106,19 @@ class UserProfileResolver {
         WHERE "user_profile"."id" = '${id}'`)
     return a[0].active
   }
+
+  @ResolveField(() => [Wiki], { nullable: true })
```
Comment (kesar) [inline src/App/userProfile.resolver.ts:110]: this piece of code its from a different issue

File: src/App/tag.resolver.ts (hunk 2 above)
Comment (AshakaE) [reply src/App/tag.resolver.ts:42]: It's not needed actually, the return wil be undefined so it falls to null

File: src/App/userProfile.resolver.ts (hunk 3 above)
Comment (AshakaE) [reply src/App/userProfile.resolver.ts:110]: yes it's from subscription API, removing it

**Review by kesar — APPROVED** [review]

**Comment by kesar** [comment]
lets see if we dont break anything in the ui :P :D

**Pull request merged**